README.md

```diff
@@ -136,122 +171,280 @@ Every resource in this repository is essential for the overall template setup. N
 
-Templates are highly configurable. This Template also showcases most of the configuration options used in action. Configurations are stored in the `package.json` file in the `generator` section.
+This Template contains an example implementation of all features includes in the Generator, both in React and in Nunjucks.
 
-### Template Specific Filters
+### React
 
-Check out [filters/mermaidDiagram.js](filters/mermaidDiagram.js) file to see an example of the custom filter that template developers can write for their template files. It exports a JavaScript function called `generateMermaidDiagram` that accepts parsed AsyncAPI document as an argument and returns a [Mermaid class diagram](https://mermaid-js.github.io/mermaid/#/) that visualizes the data model basing on schemas provided in the document.
+The list of resources that are relevant for this Template using React:
 
-Important things to notice:
-- The file contains multiple functions, but only one is exported. It means you can also have private functions and use them in your filter function to better structure your code.
-- It is good to properly validate the input passed to the filter and throw a clear error.  
-  ```js
-   if (!asyncapi || typeof asyncapi.version !== 'function') throw new Error('You need to pass entire parsed AsyncAPI document as an argument. Try this "{{ asyncapi | generateMermaidDiagram }}"');
-  ```
-  As a result, you get a helpful error during template development like:
-  ```js
-  Error: You need to pass the AsyncAPI document as an argument. Try this "{{ asyncapi | generateMermaidDiagram }}"
-  ```
-  Instead of 
-  ```js
-  TypeError: asyncapi.hasChannels is not a function
-  ```
-  It is crucial because, by default, errors that are generated from Nunjucks do not tell you where exactly you have an error in your filter. You can try to use `--debug` flag when running generation during development, but it is helpful only in some instances.
-- Parsed AsyncAPI document that you can access in the filter comes from the AsyncAPI JavaScript Parser. It provides you with extensive [API](https://github.com/asyncapi/parser-js/blob/master/API.md) that you can use to navigate through the document to extract certain parts from it, like for example:
-  ```js
-    const OPERATIONS = ['publish', 'subscribe'];
+- `template` is a directory where you keep all the files that will be processed by the Generator using React. Only files with the `.js`, `.jsx` and `.cjs` extensions are taken and processed by the Generator. The rest are skipped.
+- `partials` is a directory where you keep reusable parts (called also components) of the templates. You can also put there helper functions to handle perform actions on AsyncAPI data inside components.
```

Comment:
Shouldn't we just ignore `partials` when using React? I mean, people can call their directories the way they want and place them wherever they prefer (outside the `template` directory, of course).

Reply:
I agree, when using React partials should not be a "thing" 😄

Reply:
I removed part with `partials` but left `**NOTE**` with info where dev can create components :)